group_project/group_activity.py

```diff
@@ -1,3 +1,75 @@
+"""
+This module parses Group project XML into a tree of domain-specific objects.
+Basically it looks like the following
+
+GroupActivity (.)
+- ActivityComponent (./projectcomponent)
+-- ActivitySection (//section - different paths)
```

Comment:
Is a section currently a portion of a page when displaying a given stage? Are they all shown simunalteously when the user is in a given stage? I'm not clear on why we need sections at all, instead of putting directly questions, milestones, html content, etc. right below the stage node?